JENKINS-40472 allow blanks in parameter 'Restrict groups'

gmshake opened the pull request:
Issue link [https://issues.jenkins-ci.org/browse/JENKINS-40472](https://issues.jenkins-ci.org/browse/JENKINS-40472)

oleg-nenashev commented on src/main/java/de/theit/jenkins/crowd/CrowdConfigurationService.java:
```diff
@@ -110,9 +110,9 @@ public CrowdConfigurationService(String pGroupNames, boolean pNestedGroups) {
 		for (String group : pGroupNames.split(",")) {
 			if (null != group && group.trim().length() > 0) {
```
I would rather trim it once before this call

gmshake commented:
@oleg-nenashev @pingunaut  Done :)

KostyaSha commented:
removed null check may lead to NPE?

gmshake commented:
@KostyaSha 
Does not need null check here since tokens return from String.split(String) would never be null in this use case.
```java
for (String group : pGroupNames.split(",")) {
...
}
```
 Tested with Oracle JDK 1.7 and Oracle JDK 1.8.

gmshake commented:
@pingunaut As the master branch changed a lot since release 1.8, I'd suggest drop minor release 1.9.0 and do a major release 2.0.0
